src/manage_migrators_219_to_220_names.h

```diff
@@ -383,10 +383,10 @@ static const gchar *migrate_219_to_220_names[][2]
       "1.3.6.1.4.1.25623.1.0.96180:0:entry:timeout" },
     { "1.3.6.1.4.1.25623.1.0.96180:checkbox:List all and not only the first 100 entries",
       "1.3.6.1.4.1.25623.1.0.96180:1:checkbox:List all and not only the first 100 entries" },
-    { "1.3.6.1.4.1.25623.1.0.96180:checkbox:Install hash test Programm on the Target",
-      "1.3.6.1.4.1.25623.1.0.96180:2:checkbox:Install hash test Programm on the Target" },
-    { "1.3.6.1.4.1.25623.1.0.96180:checkbox:Delete hash test Programm after the test",
-      "1.3.6.1.4.1.25623.1.0.96180:3:checkbox:Delete hash test Programm after the test" },
+    { "1.3.6.1.4.1.25623.1.0.96180:checkbox:Install hash test Program on the Target",
```

Comment:
AFAIK these are typos in that specific VT which currently can't be changed as long as a migration to GOS 6.0 is still supported.